README.md

```diff
@@ -52,12 +52,40 @@ The app is currently hosted on Microsoft Azure. Specifically:
 * Azure Database for PostGreSQL flexible server
 * Azure Cognitive Services (for Translation)
 
-To deploy your own instance, follow the [tutorial for Flask app + PostGreSQL deployment](https://docs.microsoft.com/en-us/azure/app-service/tutorial-python-postgresql-app) but using this app instead of the sample app.
+To deploy your own instance, follow these steps:
+
+1. Sign up for a [free Azure account](https://azure.microsoft.com/free/?WT.mc_id=python-79461-pamelafox)
+2. Install the [Azure Dev CLI](https://learn.microsoft.com/azure/developer/azure-developer-cli/install-azd?WT.mc_id=python-79461-pamelafox). (If you open this repository in Codespaces or with the VS Code Dev Containers extension, that part will be done for you.)
+3. Initialize a new `azd` environment:
+
+    ```shell
+    azd init
+    ```
+
+    It will prompt you to provide a name (like "flask-app") that will later be used in the name of the deployed resources.
+
+4. Provision and deploy all the resources:
+
+    ```shell
+    azd up
+    ```
+
+    It will prompt you to login, pick a subscription, and provide a location (like "eastus"). Then it will provision the resources in your account and deploy the latest code. If you get an error with deployment, changing the location (like to "centralus") can help, as there may be availability constraints for some of the resources.
+
+5. When azd has finished deploying, you'll see an endpoint URI in the command output. Visit that URI and you should see the quiz! 🎉
+
+6. When you've made any changes to the app code, you can just run:
+
+    ```shell
+    azd deploy
+    ```
+
+Alternatively, follow the [tutorial for Flask app + PostGreSQL deployment](https://docs.microsoft.com/en-us/azure/app-service/tutorial-python-postgresql-app) but using this app instead of the sample app.
```

Comment:
```suggestion
Alternatively, follow the [tutorial for Flask app + PostGreSQL deployment](https://docs.microsoft.com/azure/app-service/tutorial-python-postgresql-app) but using this app instead of the sample app.
```